### PR TITLE
added some extra checks of round number returned from endpoint

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -86,7 +99,7 @@ impl<'a, T: Transport> DrandClient<'a, T> {
                         &self.chain_info.public_key,
                         &beacon,
                     )
-                    .map_err(|_| DrandClientError::FailedVerification)?;
+                        .map_err(|_| DrandClientError::FailedVerification)?;
```
Comment (AnomalRoil) [inline src/lib.rs]: is this really the linter suggestion? seems misaligned

**Review by AnomalRoil — APPROVED** [review]
lgtm

File: src/lib.rs (hunk 1 above)
Comment (CluEleSsUK) [reply src/lib.rs]: aye, the formatter sends it there!

Comment (CluEleSsUK) [reply src/lib.rs]: no wait... it didn't. Weirdly the RustRover formatter sends it there, but `cargo fmt` does not fuuuu

**Pull request merged**